aws-greengrass-testing-features/aws-greengrass-testing-features-cloudcomponent/src/main/resources/greengrass/components/recipes/hello_world_recipe_multiplatform.yaml

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+---
+RecipeFormatVersion: 2020-01-25
+ComponentName: com.aws.HelloWorld
+ComponentVersion: '1.0.2'
+ComponentDescription: Hello World Cloud Component.
+ComponentPublisher: Amazon
+Manifests:
+- Platform:
+    os: windows
+  Artifacts:
+    - URI: classpath:/greengrass/components/artifacts/aws-greengrass-testing-features-cloudcomponent.zip
```

Comment:
Required only to satisfy OTF, actually not used.

Reply:
Are you saying that artifacts are required for OTF, or what are you saying?

Reply:
Artifacts should not be needed

Reply:
Currently OTF  throw a NPE  when Manifests item has no Artifacts. I noted on code but it is another bug. I don't know resolve it or leave as OTF feature.

Reply:
So I just provide existing artifact to test main fix.

Reply:
That's not a feature. It should be fixed very easily.